.rubocop.yml

```diff
@@ -0,0 +1,52 @@
+AllCops:
+  Exclude:
+    - "Guardfile"
+    - "Rakefile"
+
+  DisplayCopNames: true
+
+Layout/LineLength:
+  Max: 120
+Metrics/MethodLength:
+  Max: 35
```

Comment:
- [ ] Please fix - On the Microverse linters [link](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml) the MethodLength is 20, please put it back and try to fix the error without modifying the `.rubocop.yml` file.